src/mrack/providers/utils/testcloud.py

```diff
@@ -80,6 +81,11 @@ async def create(
             args.extend(["--no_graphics"])
         if kwargs.get("keep"):
             args.extend(["--keep"])
+        if kwargs.get("ssh_path"):
```

Comment:
Isn't ssh_path used only for CoreOS instances?  AKA, does it help us somehow?

Reply:
yes that is correct we just enabled support, unfortunately it would not help us